azure-nextgen-py-appservice-docker/Pulumi.yaml

```
@@ -0,0 +1,8 @@
+name: azureng-py-appservice-docker
+runtime: python
+description: Creates Azure App Service and Deploys Docker images to
+template:
+  config:
+    location:
+      description: Azure region to deploy resources to
+      secret: false
```

Should we add a default value?

```suggestion
      default: WestUS

```